### DISCUSSION TITLE
Initial Build Failed

### DISCUSSION THREAD

**Discussion by amanrique1** [osmandapp/OsmAnd#11682] [Q&A]
build.grable is failing for Android. I tried using this documentartion
https://docs.osmand.net/en/main@latest/development/build-osmand/how-to-compile-the-android-version but it didn't work.
I'm always getting error.
![2021-05-14_10h38_08](https://user-images.githubusercontent.com/42351100/118294628-868bad80-b4a0-11eb-9946-9795a19b96ed.png)

**Comment by vshcherb** [comment]
It's probably that you have 'space' in the path.

**Reply by amanrique1** [reply]
I don't have any space

**Reply by vshcherb** [reply]
This ```cd android/OsmAnd && ../gradlew assembleNightlyFreeLegacyFatDebug``` command should work fine. Probably you don't have correct checkout of repositories - https://docs.osmand.net/en/main@latest/development/build-osmand/setup-the-dev-environment
